File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.py
N = int(input())
size = list(map(int, input().split()))
T, P = map(int, input().split())
muk = 0
for cloth in size:
    if cloth % T == 0:
        muk += cloth // T
    else:
        muk += cloth // T + 1

penMuk = N // P
penEx = N % P

print(f"{muk}\n{penMuk} {penEx}")

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int[] size = new int[6];
        for (int i = 0; i < 6; i++) {
            size[i] = sc.nextInt();
        }
        int T = sc.nextInt();
        int P = sc.nextInt();
        sc.close();
        int muk = 0;
        for (int cloth : size) {
            if (cloth % T == 0) {
                muk += cloth / T;
            } else {
                muk += cloth / T + 1;
            }
        }

        int penMuk = N / P;
        int penxEx = N % P;
        System.out.println(muk);
        System.out.println(penMuk + " " + penxEx);
    }
}